### PR TITLE
40 write route for viewing a single neighborhood

### PR DESCRIPTION
**Pull request by RobertRodes**
I think this one might be ready to close.

### CODE REVIEW

**Review by MaxHass12 — COMMENTED** [review]
It looks perfect. (apart from very minor things discovered by `nom run lint`). Approving.

**Comment by radu-constantin** [comment]
> It looks perfect. (apart from very minor things discovered by `nom run lint`). Approving.

Great job Bob! Works fine!
I also confirm what Shwetank said above and I want to add that it would be helpful to install the eslint VSCode extension so you can better see linting errors.
Also, I noticed that the 'GET /neighborhoods/id invalid id returns expected error' test is outside the `describe` block.
These are not critical issues so I will approve the pull request and we can fix them later.

File: src/tests/neighborhood_api.test.ts [hunk 1]
```diff
@@ -26,6 +26,22 @@ describe('When neighborhoods already exist in the db', () => {
...
+    expect(response.status).toEqual(200);
+    expect(response.body.id).toEqual(id);
+  });
+});
+
+test('GET /neighborhoods/id invalid id returns expected error', async () => {
+  const response = await api.get(`/api/neighborhoods/0`);
+  expect(response.status).toEqual(404);
+  expect(response.body.error).toEqual('No Neighborhood found');
 });
```
Comment (radu-constantin) [inline src/tests/neighborhood_api.test.ts:45]: This should be inside a describe block

**Review by radu-constantin — APPROVED** [review]

File: src/tests/neighborhood_api.test.ts
Comment (nina-dk) [inline src/tests/neighborhood_api.test.ts:1]: Shouldn't these tests be in their own `describe` block?

**Review by nina-dk — COMMENTED** [review]
I'm also getting some minor ESLint complaints. Other than that, and what I've mentioned about the tests, looking good 👌🏻

File: src/tests/neighborhood_api.test.ts
Comment (RobertRodes) [reply src/tests/neighborhood_api.test.ts:1]: I've moved my test into the `describe` block, and resolved conflict.

File: src/controllers/neighborhoods.ts
Comment (nina-dk) [inline src/controllers/neighborhoods.ts:1]: Is the route duplicated or am I missing something?

**Review by nina-dk — CHANGES_REQUESTED** [review]

File: src/controllers/neighborhoods.ts
Comment (radu-constantin) [reply src/controllers/neighborhoods.ts:1]: Just checked it out, you're right, it is duplicated.

Comment (RobertRodes) [reply src/controllers/neighborhoods.ts:1]: Ok, now I see it. So, we should create an issue to remove the duplicate route?

Comment (radu-constantin) [reply src/controllers/neighborhoods.ts:1]: Exactly!

Comment (RobertRodes) [reply src/controllers/neighborhoods.ts:1]: > Exactly!

All right. I will make the modification and push it.

**Comment by RobertRodes** [comment]
Ok, I think this one's done.

**Pull request merged**